django/contrib/contenttypes/management.py

```diff
@@ -82,3 +85,9 @@ def update_contenttypes(app_config, verbosity=2, interactive=True, using=DEFAULT
         else:
             if verbosity >= 2:
                 print("Stale content types remain.")
+
+
+def remove_contenttypes(app):
```

Comment:
How about making `app` an optional argument. If not specified, it would run on all `app_configs`? I guess `remove_contenttypes` is a misleading name because it will create content type too. If we had a separate `utils.py` file, then we could us the same `update_contenttypes` name. IMO, code in `management.py` is meant for use by management commands, not for user code.